packages/cozy-client/src/CozyClient.js

```diff
@@ -90,6 +90,9 @@ const securiseUri = uri => {
 
   return uri
 }
+const resolveToValue = fnOrValue => {
+  return typeof fnOrValue === 'function' ? fnOrValue() : fnOrValue
+}
```

Comment:
This function is already defined [here](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/hooks/useQuery.js#L11), some mutualization would be nice